CONTRIBUTING.md

```diff
@@ -25,6 +25,34 @@ Even better: You could submit a pull request with a fix / new feature!
    developers, or if you do not have permission to do that, you may request
    the second reviewer to merge it for you.
 
+## Release process
+
+To make it possible to automatically generate changelog, all tickets/issues must have a milestone and at least one label.
+
+A changelog can be generated using [`github-changelog-generator`][github-changelog-generator].<sup>(1)</sup>
```

Comment:
```suggestion
A changelog can be generated using the [`github-changelog-generator`][github-changelog-generator].<sup>(1)</sup>
```

Reply:
Also: as the `Footnotes` section has a header - could we make the `(1)` a link to that section ?

(same for `(2)` further below)

Reply:
I never thought of that. Love it! Will implement.